frontend/src/reducers/tables.js

```diff
@@ -326,6 +326,13 @@ const reducer = produce((draftState, action) => {
       const { id, active } = action.payload;
 
       draftState[id].activeSort = active;
+      return;
+    }
+    case types.SET_SORT_FIELDS: {
+      const { id, fields } = action.payload;
+      draftState.headers = {};
```

Comment:
Shouldn't this be in the `tables[tableId].headers` ? Or even with a more meaningful name like `sortOptions` or something ?